[BUG] README installation steps produce error

**Describe the bug**
The README's installation instructions are not accurate and produce an `npm` error.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow **Advanced (Developer experience)** instructions for local dev environment setup.
2. `npm install` will produce:
```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/meteor-node-stubs/readable-stream@git+https:/github.com/meteor/readable-stream.git#2e9112d7d31a2af6e0682db0e18679b1e5fd4694" as it does not contain a package.json file.
```

**Expected behavior**
Installing dependencies from a fresh repo should correctly install dependencies without error. 

**Fix**
Update `README` to list correct installation instructions using `meteor`.

--- a/README.md
+++ b/README.md
@@ -55,10 +55,12 @@ Additionally, we now have a `headshots` folder for hosting prosecutor pictures. 
 ### Advanced (Developer experience)
 To help build the app:
 
-1. cd to repo/app
-2. `npm install`
-3. `meteor run`
-4. `meteor mongo`
+1. install `npm` ([see official docs](https://www.npmjs.com/get-npm))
+2. install `meteor` ([see official docs](https://www.meteor.com/install))
+3. `cd` into repo's `app` folder
+4. `meteor npm install`
+5. `meteor run`
+6. `meteor mongo`
 
 Production: https://us-prosecutor-database.herokuapp.com/
 

